packages/internals/src/get-generators/generatorResolvers/prisma-client-js/check-dependencies/resolve.ts

```diff
@@ -1,6 +1,14 @@
 import path from 'path'
 import { default as _resolve } from 'resolve'
 
+/**
+ * Alternative to resolve that returns undefined instead of throwing. It also
+ * enables preserving symlinks, which is not possible with the original resolve.
```

Comment:
```suggestion
 * Alternative to 'resolve' that returns undefined instead of throwing. It also
 * enables preserving symlinks, which is not possible with the original 'resolve'.
```